GVRf/Framework/src/org/gearvrf/GVRContext.java

```diff
@@ -637,7 +668,22 @@ public AiNode wrapSceneNode(Object parent, Object matrix,
         // Scene Object
         this.recurseAssimpNodes(assimpImporter, assetRelativeFilename, wholeSceneObject,
                 rootNode, wrapperProvider);
-
+        
+        int []wrapmodes=assimpImporter.getTextureWrapMode();
+        
+         /*
+          *  if it is null that mean its a not fbx file 
+         	GVRf has only support for S and T axis but wrapmodes[] has data for 3 axes
+         
+         */
+		 if(wrapmodes!=null) 	
+         {
+        	 GVRContext context=assimpImporter.getGVRContext();
+        	 context.DEFAULT_TEXTURE_PARAMETERS.setWrapSType(WrapmodeMap.get(wrapmodes[GVRTextureWrapAxis.WrapS.getWrapAxis()]));
+        	 context.DEFAULT_TEXTURE_PARAMETERS.setWrapTType(WrapmodeMap.get(wrapmodes[GVRTextureWrapAxis.WrapT.getWrapAxis()]));
+         }
+        
+        
         return wholeSceneObject;
     }
 
```

Comment:
I think for the codes in this file you need to do Ctrl+Shift+F to do the formatting.